nixos/tests/networking.nix

```diff
@@ -907,38 +854,31 @@ let
             ), "The IPv6 routing table has not been properly cleaned:\n{}".format(ipv6Residue)
       '';
     };
-    rename = {
-      name = "RenameInterface";
+    rename.module = if networkd then {
```

Comment:
```diff
-    rename.module = if networkd then {
+    rename.enable = networkd;
+    rename.module = {
```

Possible because of a recent addition, so requires a rebase.

Would be nice to support this with non-networkd networking as well. Is that feasible?